dae/dae/studies/variants_db.py

```diff
@@ -14,18 +14,31 @@
 from dae.backends.impala.impala_helpers import ImpalaHelpers
 from dae.backends.impala.impala_variants import ImpalaFamilyVariants
 
+from dae.gene.gene_info_config import GeneInfoConfigParser
+from dae.gene.weights import WeightsLoader
```

Comment:
consider renaming 'WeightsLoader' to 'WeightsFactory'

Reply:
This is added in TODO in gpf instance trello card